### ISSUE TITLE
exwm-workspace-switch does not work

### ISSUE THREAD

**Issue by fm4d** [ch11ng/exwm#170]
Commit https://github.com/ch11ng/exwm/commit/0863f4149062089abdadd0dd795ad17d0e73da03 broke it.

**Comment by ch11ng** [comment]
Does 0b8a373e4fd092b527b34070b6556be35983062c fix this bug (not sure if we're referring to the same problem)?

**Comment by fm4d** [comment]
No, it does not fix it + it somehow broke my xrandr setup ( blackscreen on one monitor).
I should add that I have emacs 25 compiled from master branch.

**Comment by ch11ng** [comment]
I didn't test RandR.  I'll check it later.  Somehow the commit you mentioned seems irrelevant.

**Comment by fm4d** [comment]
Well, its possible that I made a mistake when I was tracking the commit so its possible its one of the commits following https://github.com/ch11ng/exwm/commit/0863f4149062089abdadd0dd795ad17d0e73da03.

**Comment by ch11ng** [comment]
> I guess that you should try to track it down in the same way I did since we have a different setups.

In which way?

I've just test the RandR module and it seems fine.  Maybe you didn't update XELB?

**Comment by fm4d** [comment]
Well I simply performed git checkout + tested if exwm-workspace-switch works until I foud out that https://github.com/ch11ng/exwm/commit/0863f4149062089abdadd0dd795ad17d0e73da03 
is the commit that broke it. Are you able to confirm that?

I am not able to test RandR right now but I updated XELB.

**Comment by ch11ng** [comment]
> Are you able to confirm that?

No, I can't see that problem, with that commit, with Emacs built from the master branch.  Please show us more details on the problem; maybe there're something like error messages?

**Comment by fm4d** [comment]
So to be sure I recompiled the current emacs-25 master, set up a new config only containing 
```
  (use-package xelb
    :load-path (lambda () (concat "~/Git/" "xelb"))
    :pin manual)

  (use-package exwm
    :load-path (lambda () (concat "~/Git/" "exwm"))
    :demand t
    :pin manual)

  (use-package exwm-config
    :load-path (lambda () (concat "~/Git/" "exwm"))
    :demand t
    :pin manual
    :config (exwm-config-default))
```
and nothing changed, `exwm-workspace-switch` just does not do anything. With `exwm-debug-on t` there is nothing in messages or backtrace. I am running out of ideas what to test.

**Comment by ch11ng** [comment]
How did you call `exwm-workspace-switch`? I've just added a major mode menu, could you try if you can switch workspace from that?

**Comment by ch11ng** [comment]
Sorry my bad.  It was caused by two typos and is fixed in 1a716d39397edfc2a083c479817ea7d40b23fd54.
